Source/AtomicEditor/Editors/ResourceEditorEvents.h

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright (c) 2014-2015, THUNDERBEAST GAMES LLC All rights reserved
+// LICENSE: Atomic Game Engine Editor and Tools EULA
+// Please see LICENSE_ATOMIC_EDITOR_AND_TOOLS.md in repository root for
+// license information: https://github.com/AtomicGameEngine/AtomicGameEngine
+//
+
+#pragma once
+
+#include <Atomic/Core/Object.h>
+
+namespace AtomicEditor
+{
+
+EVENT(E_EDITORCLOSERESOURCE, EditorCloseResource)
+{
+PARAM(P_EDITOR, Editor);            // ResourceEditor
+PARAM(P_NAVIGATE, NavigateToAvailableResource); // bool
+}
+
+EVENT(E_EDITORRESOURCECLOSECANCELED, EditorResourceCloseCanceled)
+{
```

Comment:
Here we have:
 
EditorCloseResource
EditorResourceCloseCanceled

Probably should make them consistent so:

EditorResourceClose
EditorResourceCloseCanceled